setup.sh

```diff
@@ -69,10 +67,10 @@ make install
 echo 'PATH=$PATH:/usr/local/pfring/bin:/usr/local/pfring/sbin' >> /etc/bash.bashrc
 
 #Get BRO
-cd ../../
+cd ../../../
 git clone -b 'v2.5.1' --recursive https://github.com/bro/bro.git
 cd bro
-./configure --prefix=/usr/local/bro --with-pcap=/usr/local/pfring/lib
+./configure --prefix=/usr/local/bro --with-pcap=/usr/local/pfring
```

Comment:
The extra `/lib` was the culprit for the issue I was having.  Removing it is in line with what the official instructions show [here](https://www.bro.org/sphinx/configuration/index.html#installing-pf-ring).